app/models/concerns/dynamic_attribute.rb

```diff
@@ -7,6 +7,7 @@
 module DynamicAttribute
     extend ActiveSupport::Concern
     include Rails.application.routes.url_helpers
+    include ActionView::Helpers::DateHelper
```

Comment:
@Sushmit1997 you should include all the helper methods provided by Rails for developer happiness. Currently you're only including a subset with the `DateHelper`. It should be wider in scope.

Eg: https://apidock.com/rails/ActionView/Helpers/TextHelper/pluralize -- `pluralize` will not work. We want to make sure that client engineers have access to the amazing helper methods that come with Rails.

Reply:
see here for the helpers we need: 

1. https://api.rubyonrails.org/classes/ActionController/Helpers.html
2. https://guides.rubyonrails.org/action_view_helpers.html

Reply:
Okay, I'll add the required helpers.